cmake_modules/commonSetup.cmake

```diff
@@ -676,8 +677,13 @@ IF ("${COMMONSETUP_DONE}" STREQUAL "")
 
   IF ( NOT MAKE_DOCS_ONLY )
       IF ("${EXTERNALS_DIRECTORY}" STREQUAL "")
-        SET(bisoncmd "bison")
-        SET(flexcmd "flex")
+        IF (WIN32)
```

Comment:
@cloLN Is it possible to test with this PR still will compile the client tools on windows?